### PR TITLE
[feature](executor)make task wait timeout config

### PR DESCRIPTION
**Pull request by wangbo**
make task wait timeout config

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: be/src/common/config.cpp [hunk 1]
```diff
@@ -956,6 +956,7 @@ DEFINE_Bool(enable_debug_points, "false");
 
 DEFINE_Int32(pipeline_executor_size, "0");
 DEFINE_mBool(enable_workload_group_for_scan, "false");
+DEFINE_mInt64(wait_task_timeout_ms, "10000");
```
Comment (yiguolei) [inline be/src/common/config.cpp:959]: workload_group_scan_task_wait_timeout_ms

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by hello-stephen** [comment]
run buildall

**Pull request merged**